src/database.ts

```diff
@@ -136,7 +140,21 @@ export class Database {
     return this.#open && !this.autocommit;
   }
 
-  constructor(path: string | URL, options: DatabaseOpenOptions = {}) {
+  constructor(path: string | URL = "", options: DatabaseOpenOptions = {}) {
+    this.int64 = options.int64 ?? false;
+    this.unsafeConcurrency = options.unsafeConcurrency ?? false;
+  
+    if (!path) {
+      return;
+    }
+    this.open(path, options);
+  }
+
+  /** Open a Database. */
+  open(path: string | URL, options: DatabaseOpenOptions = {}): boolean {
```

Comment:
Unsure about returning a boolean here. I think we should simply error if user tries to open an already open Database connection object, and make the return type void to be consistent with rest of the API.

Reply:
I went ahead and changed that. I also added a TypeError in case path is empty.